### ISSUE TITLE
Extend -f Tangram.app OS X option to allow arbitrary scene YAML

### ISSUE THREAD

**Issue by nvkelso** [tangrams/tangram-es#1570]
From the OS X [README](https://github.com/tangrams/tangram-es/tree/master/platforms/osx) I see I can:

> open build/osx/bin/tangram.app/Contents/MacOS/tangram -f /path/to/your/scene.yaml

But if I want to enable Walkabout bike map I should only need to construct a little YAML snippet to pass in with the `-f` argument more like the Tangram JS workflow.

Instead I have to construct an actual file to do the same thing, and pass in that file reference instead.

See also:

- https://github.com/mapzen/website/blob/master/products/maps/walkabout-style-bike-map.html#L44-L51

```
scene: {
              import: 'https://mapzen.com/carto/walkabout-style/5/walkabout-style.zip',
              global: {
                sdk_mapzen_api_key: YOUR-API-KEY-HERE,
                sdk_bike_overlay: true,
                sdk_path_overlay: false
              }
            }
```

And the file I constructed locally to pass in instead:

```
import:
    - https://mapzen.com/carto/walkabout-style/5/walkabout-style.zip

global:
    sdk_mapzen_api_key: YOUR-API-KEY-HERE
    sdk_bike_overlay: true
    sdk_path_overlay: false
```

**Comment by tallytalwar** [comment]
Should be a follow-up or part of the work being done for: #1534

**Comment by matteblair** [comment]
Maybe this is too weird, but we could allow the demo app to accept a scene from `stdin` so you could just pipe in the scene you want:
```
"{ import: some/scene.yaml, global: { sdk_mapzen_api_key: YOUR_KEY_HERE } }" | tangram
```

**Comment by nvkelso** [comment]
That’s not crazy.

**Comment by tallytalwar** [comment]
Addressed in #1577
